### PR TITLE
551 limit sightinig fields and support new style site settings

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Limit AGS creation to only supported fields
- Added one extra EDM sighting field as used in tests
- Updated tests to reflect the changes fixing a few oddities on the way.

### CODE REVIEW

File: app/modules/sightings/parameters.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class PatchSightingDetailsParameters(PatchJSONParameters):
         '/locationId',
         '/taxonomies',
         '/verbatimLocality',
+        '/verbatimEventDate',
```
Comment (hwindsor) [inline app/modules/sightings/parameters.py:44]: @naknomum I added this as it was used in one of the integ tests. Is this correct or should we move the usage from the intreg tests?

**Review by brmscheiner — COMMENTED** [review]
does this modify the POST AGS API in a way such that existing successful requests could start failing?

**Comment by hwindsor** [comment]
> does this modify the POST AGS API in a way such that existing successful requests could start failing?

It's a modification but I tested with the FE with the fields in the API and a custom field and that worked so my testing has said that it should not start failing.

**Comment by brmscheiner** [comment]
sounds great!

File: app/modules/sightings/parameters.py (hunk 1 above)
Comment (colinwkingen) [reply app/modules/sightings/parameters.py:44]: Might as well leave this as it is present in a lot of data that will be migrated. Even though this is for new AGS we can support it.

**Review by colinwkingen — APPROVED** [review]
LGTM

**Pull request merged**